PetAdoption-iOS/PetAdoption-iOS/Views/HomeLandscapeCollectionViewLayout.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HomeLandscapeCollectionViewLayout.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 9/10/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+
+class HomeLandscapeCollectionViewLayout: UICollectionViewFlowLayout
+{
+    ////////////////////////////////////////////////////////////
+    // MARK: - Properties
+    ////////////////////////////////////////////////////////////
+
+    override var itemSize: CGSize
```

Comment:
So, if I remove the setter, I get the following error:
`Getter for 'itemSize' with Objective-C selector 'itemSize' conflicts with getter for 'itemSize' from superclass 'UICollectionViewFlowLayout' with the same Objective-C selector.`

If I take everything in the getter, and just place that in the trailing closure for the property, I get this error:
`Cannot override mutable property with read-only property 'itemSize'`

I'm not sure what to do about each of those errors.  I've seen that code used in other projects and tutorials online.  I will admit that I'm unsure of exactly how that works.  If you've got some insight, I'm all ears. :)